week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,29 @@
-// Add comments to explain what this function does. You're meant to use Google!
+/* Add comments to explain what this function does. You're meant to use Google!*/
 function getNumber() {
     return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+/*expected return: The Math.random() function returns a floating-point, pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1 in decimal form) and then multiplying by 10. 
+
+-source - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random */
+
+/* Add comments to explain what this function does. You're meant to use Google!*/
 function s(w1, w2) {
     return w1.concat(w2);
 }
 
+/*Expected return: The concat() method concatenates the string arguments to the calling string and returns a new string.
```

Comment:
This one is a little bit tricky! There is actually a `.concat()` method on strings **and** a `.concat()` method on arrays. Which one do you think we're using here?